flake8_black.py

```diff
@@ -98,6 +106,26 @@ def _file_mode(self):
             assert "got an unexpected keyword argument" in str(e), e
             return None
 
+    @classmethod
+    def add_options(cls, parser):
+        """Adding flake8-black-config option."""
+        parser.add_option(
+            "--flake8-black-config",
```

Comment:
I see no reason to include flake8 in the setting name given the usage context, how about just ``--black-config`` here?

Reply:
(Oops - Just realised I suggested the verbose name on #9, ah well - I said at the time I wasn't sure about the name)